dpctl/tensor/libtensor/include/kernels/elementwise_functions/isnan.hpp

```diff
@@ -45,6 +46,7 @@ namespace isnan
 
 namespace py = pybind11;
 namespace td_ns = dpctl::tensor::type_dispatch;
+namespace mu_ns = dpctl::tensor::math_utils;
```

Comment:
In ``isnan.hpp``, ``math_utils`` is imported and the namespace is set, but never used to call ``mu_ns::isnan``

Reply:
Pushed the fix. Thanks for spotting that out @ndgrigorian . I was after `std::isnan` instances and missed `sycl::isnan`.